packages/salesforcedx-utils-vscode/src/i18n/localization.ts

```diff
@@ -29,6 +29,7 @@ export class Localization implements LocalizationProvider {
   }
 
   public localize(label: string, ...args: any[]): string {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
what are these commented lines for?

Reply:
@diyer used to override eslint rules in code to bypass lint checks. The syntax is `// eslint-disable-next-line <rule(s) to disable>`.